src/lib/comms/sol-bluetooth-impl-bluez.c

```diff
@@ -29,6 +29,8 @@
 
 #include "sol-bluetooth-impl-bluez.h"
 
+#define AGENT_PATH "/org/solettaproject/agent"
```

Comment:
I don't see any value, and this only makes sense for the BlueZ implementation, but perhaps I am missing something.